CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `orchard::builder`:
+  - `SpendError` 
+  - `OutputsDisabled` 
+
+### Changed
+- `orchard::builder::Builder::{add_spend, add_output}` now use 
+  concrete error types instead of `&'static str`s. 
+- `orchard::builder::Error` is now `BuildError` to differentiate from
+  new error types
```

Comment:
Formatting:
```suggestion
- `orchard::builder`:
  - `Builder::{add_spend, add_output}` now use concrete error types instead of
    `&'static str`s.
  - `Error` has been renamed to `BuildError` to differentiate from new error
    types.
```